### ISSUE TITLE
Product logos displayed at the top of KB articles are missing alt text

### ISSUE THREAD

**Issue by denyshon** [mozilla/sumo#3257]
**Steps to reproduce**
1. Open any KB article, e.g., https://support.mozilla.org/kb/update-firefox-latest-release
2. Check the alt text of the product logo, displayed at the top of the article:
<img width="1607" height="395" alt="Image" src="https://github.com/user-attachments/assets/3e2efeaf-c3ba-4fbe-aec4-f4536a7ad7f5" />

**Expected behavior**
The logo has "{product} logo" alt text, like in other places, such as https://github.com/mozilla/kitsune/blob/e9732218b14152c4d49928b2fcfb69e2477e197a/kitsune/questions/jinja2/questions/includes/aaq_macros.html#L169

**Actual behavior**
The logo is missing alt text.

**Additional context**
The related code is here: https://github.com/mozilla/kitsune/blob/e9732218b14152c4d49928b2fcfb69e2477e197a/kitsune/wiki/jinja2/wiki/document.html#L91

**Comment by denyshon** [comment]
@emilghittasv, could you please mark this as good-first-bug?
